packages/pangea-sdk/PangeaCyber.Net/Share/Models/Bucket.cs

```diff
@@ -0,0 +1,22 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PangeaCyber.Net.Share.Models
+{
+    /// <summary>Bucket.</summary>
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
```

Comment:
So with this we avoid setting `JsonProperty` per object, don't we?

Reply:
Right.